### PR TITLE
Update create-wip-policy-using-intune-azure.md, clarify the scope of Azure RMS

### PR DESCRIPTION
**Pull request by MaratMussabekov**
clarify the scope of Azure RMS as discussed in issue #2724 
source  [Protect your enterprise data using Windows Information Protection (WIP)](https://docs.microsoft.com/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Hi @Justinha ,
according to [Protect your enterprise data using Windows Information Protection (WIP)](https://docs.microsoft.com/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip) :
"Finally, another data protection technology, Azure Rights Management also works alongside WIP to extend data protection for data that leaves the device, such as when email attachments are sent from an enterprise aware version of a rights management mail client."

Don't you think it would be useful to note that RMS works not only when data is copied to external drive, but also when sent as attachment?

Thank you

**Comment by lightupdifire** [comment]
@Justinha,
Would be interesting to understand the behavior with "Sent as attachment",
In our setup, if I send by mail WIP protected file, as attachment, as example to Gmail and try access from public PC, I can open file with no AzureRMS prompt.
I have 2x tenants with different setups: 1st is with Silent mode, 2nd is with Prompt. In 2nd I use Template ID. Using both tenants, sending file to Gmail not really encrypt file by AzureRMS, i dont see protection applied...

**Comment by Justinha** [comment]
@MaratMussabekov Ahhh, thanks so much I see what you mean now. Unfortunately, that text is misleading because email attachment is not actually protected. Only copy to removable storage is protected by AIP. Email attachment protection is a goal but not available today. This is more clearly stated here: https://www.microsoft.com/security/blog/2018/05/15/use-windows-information-protection-wip-to-help-make-accidental-data-leakage-a-thing-of-the-past/

So we should change the example to refer to removable media or just remove it for being unclear. I prefer to change the example from email to USB, but I appreciate your input as the reader. Would you mind submitting a PR with a change to that topic? I am happy to do it on my side also, just let me know. 

@lightupdifire Thanks for sharing what you experienced, and confirmed as by design today.

File: windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md [hunk 1]
```diff
@@ -524,7 +524,7 @@ WIP can integrate with Microsoft Azure Rights Management to enable secure sharin
 
 To configure WIP to use Azure Rights Management, you must set the **AllowAzureRMSForEDP** MDM setting to **1** in Microsoft Intune. This setting tells WIP to encrypt files copied to removable drives with Azure Rights Management, so they can be shared amongst your employees on computers running at least Windows 10, version 1703.
 
-Optionally, if you don’t want everyone in your organization to be able to share your enterprise data, you can set the **RMSTemplateIDForEDP** MDM setting to the **TemplateID** of the Azure Rights Management template used to encrypt the data. You must make sure to mark the template with the **EditRightsData** option. 
+Optionally, if you don’t want everyone in your organization to be able to share your enterprise data, you can set the **RMSTemplateIDForEDP** MDM setting to the **TemplateID** of the Azure Rights Management template used to encrypt the data. You must make sure to mark the template with the **EditRightsData** option. This template will be applied to the protected data that leaves the device (for example, by being copied to removable drive or sent as an email attachment).
```
[the remark this answers is not in the document]
Comment (mypil) [reply windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md]: @MaratMussabekov - please check on the recommendation from Justinha and TokyoScarab and do the needful. Thank you.

[the remark this answers is not in the document]
Comment (Justinha) [reply windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md]: Indeed, if you would be kind enough to edit pull request, I think we want this change in the last sentence:
Optionally, if you don’t want everyone in your organization to be able to share your enterprise data, you can set the **RMSTemplateIDForEDP** MDM setting to the **TemplateID** of the Azure Rights Management template used to encrypt the data. You must make sure to mark the template with the **EditRightsData** option. This template will be applied to the protected data that is copied to a removable drive.

**Comment by MaratMussabekov** [comment]
Hello @Justinha , @TokyoScarab ,
Thank you for your recommendations, I've applied another one commit, could you please check?
Thank you

**Pull request merged**